sashimi-webapp/src/logic/documentPackager/documentFormatter.js

```diff
@@ -0,0 +1,64 @@
+/*
+** HTML XSS Filtering plugin by leizongmin
+** https://github.com/leizongmin/js-xss
+**
+** Features:
+** 1. Specifies HTML tags and their attributes allowed with whitelist
+** 2. Handle any tags or attributes using custom function.
+*/
+import xssFilter from 'xss';
+
+// get a copy of default whiteList
+const whiteList = xssFilter.getDefaultWhiteList();
+
+/*
+** Custom whitelist (Adding to current whiteList to prevent filtering
+** out some required HTML attributes for styling by plugins)
+*/
+whiteList.pre.push('class');
+whiteList.code.push('class');
+whiteList.span.push('class', 'aria-hidden', 'style', 'role');
+
+// New tags not in whiteList that are required by plugins
```

Comment:
As discussed during meeting, maybe we can use of the `onIgnoreTag` for future unknown tag included by plugins.

Ref: https://github.com/leizongmin/js-xss#customize-the-handler-function-for-tags-not-in-the-whitelist